vault/tests/common.py

```diff
@@ -11,6 +11,7 @@
 COMPOSE_FILE = os.path.join(HERE, 'docker', 'docker-compose.yaml')
 HOST = get_docker_hostname()
 PORT = '8200'
+CMD_USE_SUDO = os.environ.get('CMD_USE_SUDO')  # set CMD_USE_SUDO=1 to run commands with sudo
```

Comment:
Can we use https://github.com/DataDog/integrations-core/blob/d4ffe02013761039242a0263c86876b2a02a3b65/datadog_checks_dev/datadog_checks/dev/utils.py#L34-L35?